book/website/communication/peer-review/peer-review-guidance.md

```diff
@@ -24,10 +24,11 @@ Are you planning to submit your own work there?
 Is this journal an [Open Access](https://the-turing-way.netlify.app/reproducible-research/open/open-access.html) journal?
 Beware of [predatory journals](https://predatoryjournals.com/journals/).
 
-1. Check if you are the right person to review the paper. 
+1. Check if this paper is of interest to review and if you are the right person to review the paper. 
 Reviewers do not need to be experts in all respects. 
 It is a good practice to describe your disciplinary background and expertise in relation to the manuscript you are invited to review.
 If you do not have useful expertise to improve the paper, politely decline.
+If you do not have sufficient time to review the paper, politely decline. 
 
```

Comment:
Recommending other potential reviewers when declining is helpful, but not necessary -- a timely accept/decline response to the review request is more important.

Reply:
Added, thanks!